source/develop/release-management/features/gluster/gluster-reset-brick-support.html.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Gluster Reset Brick Support
+category: feature
+authors: godas
+feature_name: Gluster management
+feature_modules: engine,gluster
+feature_status: Completed
+---
+
+# Gluster Support
+
+## Summary
+
+This feature provides support for reset-brick Gluster based storage clusters in oVirt. Glusterfs repo is available at <http://download.gluster.org/pub/gluster/glusterfs/>
+
+## Owner
+
+*   Feature owner: Gobinda Das <godas@redhat.com>
+    -   GUI Component owner: Gobinda Das <godas@redhat.com>
+    -   Engine Component owner: Gobinda Das <godas@redhat.com>
+    -   VDSM Component owner: Gobinda Das <godas@redhat.com>
+    -   QA Owner: SATHEESARAN <sasundar@redhat.com>
+
+## Current Status
+
+*   Requires : glusterfs >= 3.9
+
+## Detailed Description
+
+This feature will introduce the capability of reset existing brick from the oVirt UI. Administrator will be able to perform Gluster volume reset-brick operation.
```

Comment:
It would be good to mention when reset brick should be called - i.e for what use cases?

Reply:
Done